logging/docs/conf.py

```diff
@@ -0,0 +1,310 @@
+# -*- coding: utf-8 -*-
```

Comment:
I thought I had asked about this when it showed up in `error_reporting`:  how does this `logging/docs` directory work with the one in the top-level directory?  Are we just prepping for repo-ectomy?

Reply:
Basically pre-paring for repoectomy, but at this point I'm not paying a whole lot of attention to this.